### ISSUE TITLE
WebGl issues on Windows 10

### ISSUE THREAD

**Issue by gender404** [avocode/avocode#1877]
---
* **Version**: 2.19.2
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 76438
---
```
TypeError: Cannot read property 'deinitializeOrchestrator' of null
    at t.finalize (C:\Users\jdallas\AppData\Local\avocode\app-2.19.2\resources\app.asar\node_modules\@avocode\stage\dist\stage.js:1:11407)
    at e.value (file:///C:/Users/jdallas/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3383584)
    at e.value (file:///C:/Users/jdallas/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3383141)
    at destroy (file:///C:/Users/jdallas/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:3740486)
    at Immediate.<anonymous> (file:///C:/Users/jdallas/AppData/Local/avocode/app-2.19.2/resources/app.asar/dist/avocode.min.js:1:1219779)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

**Comment by vojtatranta** [comment]
Aha, that's a bug we accidentally released in `2.19.2` for a second. Then we rolled back and it managed to get to you. Please download proper `2.19.2` version here: https://avocode.com/download

**Comment by gender404** [comment]
Well that still doesn’t work.

Are you planning to push a 2.19.3 hot fix to resolve this issue?

[cid:image001.png@01D3AA2B.3C564A00]



Jonathan Dallas
Senior Front End Developer

VELIR
p: 617.440.3837
c: 781.526.6237
f: 617.812.0057
e: jonathan.dallas@velir.com<mailto:jonathan.dallas@velir.com>
w: www.velir.com<http://www.velir.com/>
